code/dataaccess/schema.q

```diff
@@ -1,7 +1,7 @@
 \d .schema
 
-// load in tickerplant schema to determine default time column
-schema:.proc.loadf[getenv[`TORQAPPHOME],"/database.q"];
+// define the tickerplant to cross check for default timecolumn
+tickerplant:`stp1
```

Comment:
I don't think in the dataaccess codebase we should have a dependency on a tickerplant.
If we need to load a schema file, it should be defined separately as a separate piece of config

Reply:
All done. However, if the user prefers to not fill out the config it'll connect to the stp as a last resort.

Reply:
The dataaccess code should be completely independent from any tickerplant related schemas. For example, if we have the use case where we don't have a tickerplant (because it is all historic data loaded at T+1) then what is the fallback? I think we should only have three types of parameters:

1. required (must be supplied)
2. required with a default/fallback option (does not need to be supplied)
3. optional (does not need to be supplied)

If the paramter is of type 1 (which I think is what you are saying here) then if it isn't supplied the functionality should fail on initialisation e.g. something like 

q).dataaccess.init[]
'failed to initialise as time column not supplied 

or something simiar, I haven't looked into the syntax. If init is successful, it should return a 1b.